src/coreclr/jit/assertionprop.cpp

```diff
@@ -787,6 +791,10 @@ void Compiler::optPrintAssertion(AssertionDsc* curAssertion, AssertionIndex asse
         vnStore->vnDump(this, curAssertion->op1.bnd.vnLen);
         printf("]");
     }
+    else if (curAssertion->op1.kind == O1K_VN)
+    {
+        printf("[" FMT_VN "]", curAssertion->op1.vn);
```

Comment:
Use `vnStore->vnDump`?

Reply:
will address in https://github.com/dotnet/runtime/pull/110171